src/components/BookInfo/BookDetails/BookDetails.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FunctionComponent } from "react";
+import { TextContent } from "../../global/TextContent/TextContent";
+import { BookDetailsProps } from "./interfaces/BookDetailsProps";
+import styles from "./BookDetails.module.scss";
+import moment from "moment";
+import { IndustryIdentifier } from "../../BookFinder/BookList/interfaces/IndustryIdentifier";
+
+export const BookDetails: FunctionComponent<BookDetailsProps> = (props) => {
+  const renderIndustryIdentifiers = (
+    industryIdentifiers: IndustryIdentifier[]
+  ) => {
+    return industryIdentifiers
+      .filter(
+        (industryIdentifier: IndustryIdentifier) =>
+          industryIdentifier.type === "ISBN_10" ||
+          industryIdentifier.type === "ISBN_13"
+      )
+      .map((industryIdentifier: IndustryIdentifier) => {
+        return (
+          <React.Fragment key={industryIdentifier.identifier}>
+            <TextContent className={styles.label} fontSize={12}>
+              {industryIdentifier.type.replace("_", "-")}
+            </TextContent>
+            <TextContent fontSize={18}>
+              {industryIdentifier.identifier}
+            </TextContent>
+          </React.Fragment>
+        );
+      });
+  };
+
+  const renderOtherCategories = (categories: string[]) => {
+    categories.shift();
+    return categories.map((category: string) => {
+      return (
+        <TextContent key={category} className={styles.badge}>
+          {category}
+        </TextContent>
+      );
+    });
+  };
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.categoriesWrapper}>
+        {renderOtherCategories(props.book.volumeInfo.categories)}
+      </div>
+      <TextContent className={styles.label} fontSize={12}>
+        Published by
+      </TextContent>
+      <TextContent fontSize={18}>{props.book.volumeInfo.publisher}</TextContent>
+      <TextContent className={styles.label} fontSize={12}>
+        Publishing date
+      </TextContent>
```

Comment:
Sviđa mi se ovo, clean podosta :) Kako ti se čini kontrola nad fontSize ovako unutar propova? To sam smislio za p projekt jer smo često imali mnoge različlitosti kod fontSize-a članaka/tablica itd. Ukoliko koristiš često ili samo 18 i 12 veličine, možeš si uvijek napraviti komponente za ta dva fontSize, ovako imaš veću fleksibilnost tho.

Reply:
na prvu mi je bilo cudno zakaj fontsize, al sam onda skuzil da bi za svaki taj moral radit klasu (ak ocu neki drugaciji font-size) pa sam odjebal to skroz, hahah
sad mi se svida, super mi je rjesenje